pkg/sbom/spdx/marshal_test.go

```diff
@@ -716,6 +758,17 @@ func TestMarshaler_Marshal(t *testing.T) {
 			spdxDoc, err := marshaler.Marshal(tc.inputReport)
 			require.NoError(t, err)
 
+			sort.Slice(spdxDoc.Packages, func(i, j int) bool {
+				return spdxDoc.Packages[i].PackageSPDXIdentifier < spdxDoc.Packages[j].PackageSPDXIdentifier
+			})
+			sort.Slice(tc.wantSBOM.Packages, func(i, j int) bool {
+				return tc.wantSBOM.Packages[i].PackageSPDXIdentifier < tc.wantSBOM.Packages[j].PackageSPDXIdentifier
+			})
+
+			if diff := cmp.Diff(tc.wantSBOM, spdxDoc,
```

Comment:
I'm curious why we need go-cmp here. Doesn't `assert.Equal` work?

Reply:
done